pages/governance/pipeline_templates.md

```diff
@@ -28,35 +28,35 @@ An administrator can test a pipeline template against a pipeline using the _New
 
 If a template exists for the organization, it can be selected from the _Pipeline template_ dropdown to create a new build using the step configuration from that template.
 
-## Assigning a pipeline template to a pipeline
+## Requiring pipeline templates
 
-To permanently override a pipeline's step configuration with a pipeline template, administrators must first require pipeline templates via settings (see next section).
+The power of pipeline templates comes from how much you require their use. Administrators can select from the following options, listed in increasing strictness:
 
-Once pipeline templates are required, there are three options for assigning a template to individual pipelines:
+1. **Do not require pipeline templates:** Pipeline steps remain editable for any user with permission to create or update a pipeline. Templates can be tested against pipelines but cannot be assigned to them.
```

Comment:
I wrote the sentence I'm suggesting modifying here, but upon re-reading I'm wondering if a clarification like this makes sense?

```suggestion
1. **Do not require pipeline templates:** Pipeline steps remain editable for any user with permission to create or update a pipeline. Templates can be tested (by administrators) against pipelines but cannot be assigned to them.
```